web/frontend/src/components/slam-2d-render.vue

```diff
@@ -161,16 +288,23 @@ const updateCloud = (pointcloud: Uint8Array) => {
     }
   }
 
-  scene.add(points);
-  scene.add(marker);
-  scene.add(intersectionPlane);
-};
+  // construct grids
+  const axesHelper1 = createAxisHelper('AxesPos', Math.PI / 2)
+  const axesHelper2 = createAxisHelper('AxesNeg', -Math.PI / 2)
+  const gridHelper = createGridHelper(points);
+
+  // add objects to scene
+  scene.add(
+    axesHelper1,
+    axesHelper2,
+    gridHelper,
+    points,
+    intersectionPlane
+  );
 
-const updatePose = (newPose: commonApi.Pose) => {
-  const x = newPose.getX();
-  const z = newPose.getZ();
-  marker.position.setX(x);
-  marker.position.setZ(z);
+  if (props.pose !== undefined) {
+    updatePose(props.pose!);
```

Comment:
Why do we need to do this? `updatePointCloud` updates when the pointcloud updates, not when the pose updates. Why do we need to update the pose?

Reply:
what happens is 
onMount, watch for pose is triggered, watch for pointcloud is triggered.
since the last thing that happens is the `updatePointCloud` it would put the points on top of the marker making it unseeable.

Reply:
Is there no way to control the order of rendering so we don't need to constantly re render static entities?

Reply:
deferring to @micheal-parks

Reply:
@micheal-parks ?

Reply:
any update on this?

Reply:
@nfranczak can we please file a ticket on this so we know to come back to it before we consider 2d motion planing for bases complete?

Reply:
done.

https://viam.atlassian.net/browse/RSDK-2960